_functions.sh

```diff
@@ -1346,17 +1389,87 @@ do_deploy() {
       exit 1
     ;;
   esac
-  # Hack 
-  do_configure_chat
-  do_configure_apache
-  do_create_deployment_descriptor
-  echo_info "Server deployed"
+}
+
+do_docker_server_deploy(){
+  # We search the server directory
```

Comment:
looks a lot of duplicated code just to avoid 2 mkdirs

Reply:
The duplicated code is here to do separation between the classical and the docker deployment. It is here to do not broke something on the existent code